tests/src/test/fdc3.broadcast.ts

```diff
@@ -1,126 +1,181 @@
-import { Listener, Channel, Context, ContextTypes } from "@finos/fdc3";
+import { Listener, Channel, Context, getCurrentChannel } from "@finos/fdc3";
 import { assert, expect } from "chai";
 import constants from "../constants";
-import fdc3AddContextListener from "./fdc3.addContextListener";
 import APIDocumentation from "../apiDocuments";
 
 const documentation =
   "\r\nDocumentation: " + APIDocumentation.desktopAgent + "\r\nCause:";
+let timeout: NodeJS.Timeout;
```

Comment:
```suggestion
let timeout: number;
```
Common bad type suggestion as setTimeout on node has a different type to the browser version. If you have any trouble with it where used, apply casting:

```ts
let n: number;
n = setTimeout(function () { /* snip */  }, 500) as unknown as number;
```